docs/SNAPTEL.md

```diff
@@ -103,7 +111,7 @@ Example Usage
 
 In one terminal window, run snapteld (log level is set to 1 and signing is turned off for this example):
 ```
-$ $SNAP_PATH/bin/snapteld -l 1 -t 0
+$ snapteld -l 1 -o '' -t 0
```

Comment:
This works because we are overriding anything set explicitly in a config with '' and there is a check somewhere that if the value is empty we go to stdout?

Also does it make sense to be explicit in the arguments here so that it's more clear what is going on?